### ISSUE TITLE
GCop 653

### ISSUE THREAD

**Issue by marjanjavid** [Geeksltd/GCop#268]
no warning is kicked

**Comment by marjanjavid** [comment]
I wrote a unit test for this rule on https://gitlab.com/Paymon/GCop/commit/cddc7fb8222a5974a59266408dea466e7d388847

There is a problem about this rule
suppose we have a code like below 
```csharp
var res= DelayAsync();
var resValue = res.Result;
```
in the above code we should see warning, but nothing is kicked.

**Comment by marjanjavid** [comment]
There is a problem about this rule
suppose we have a code like below 
```csharp
var res= DelayAsync();
var resValue = res.Result;
```
in the above code we should see warning, but nothing is kicked.
